deployments/Dockerfiles/tendermint/tendermint.py

```diff
@@ -30,9 +30,10 @@
 from typing import Any, Dict, List, Optional
 
 
+_TCP = "tcp://"
 ENCODING = "utf-8"
-DEFAULT_P2P_LISTEN_ADDRESS = "tcp://0.0.0.0:26656"
-DEFAULT_RPC_LISTEN_ADDRESS = "tcp://0.0.0.0:26657"
+DEFAULT_P2P_LISTEN_ADDRESS = f"{_TCP}0.0.0.0:26656"
+DEFAULT_RPC_LISTEN_ADDRESS = f"{_TCP}0.0.0.0:26657"
```

Comment:
Why create a constant for that and not also for `0.0.0.0` for example? Such as in:

https://github.com/valory-xyz/open-autonomy/blob/51c6881b10fb44f7e3273cf90f5366d268b66c93/packages/valory/connections/abci/connection.py#L99

Reply:
not relevant, it's a remnant from earlier introduction of `grpc://` as a prefix. Would be nice though to collect all of this in one place and using something like [urlparse](https://docs.python.org/3/library/urllib.parse.html)